addons/mass_mailing/tests/test_mailing_list.py

```diff
@@ -334,3 +334,27 @@ def test_mailing_update_optout(self):
             [('contact_id', '=', contact.id)]
         )
         self.assertEqual(subs.list_id, ml_1 + ml_2)
+
+
+@tagged('mailing_list')
+class TestMassMailingActions(MassMailCommon):
```

Comment:
Hm, not a "mailing_list" test so it should neither be tagged nor in this file. Maybe put it in "test_mailing_ui"? Seems relevant enough